sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/SchemaTranslation.java

```diff
@@ -65,7 +67,7 @@
 })
 public class SchemaTranslation {
 
-  private static final String URN_BEAM_LOGICAL_DECIMAL = "beam:logical_type:decimal:v1";
+  private static final String URN_BEAM_LOGICAL_DECIMAL = FixedPrecisionNumeric.BASE_IDENTIFIER;
```

Comment:
Could you add this in schema.proto and reference that here? Then we can document the format in schema.proto as well

Reply:
added "beam:logical_type:decimal:v1" into schema.proto